brainbeats-backend/Controllers/TestController.cs

```diff
@@ -16,14 +15,13 @@ public class TestController : ControllerBase
     [HttpPost]
     [Route("create")]
     public async Task<IActionResult> CreateSeed(dynamic req) {
```

Comment:
What exactly is this test controller testing?

Reply:
Fixed: TestController is renamed to SeedController to better represent its purpose: seeding data into the database